tolgee-cli/push-pull-strings.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: push-pull-strings
+title: Pushing & pulling strings
+---
+
+import Experimental from './components/experimental.mdx'
+
+<Experimental/>
+
+Pushing and pulling strings from Tolgee is the most basic way to interact with strings. Similarly to a Git project, it
```

Comment:
Similarly `as with a` Git project - or something similar, but this formulation is a bit confusing